### DISCUSSION TITLE
Make available via VCPKG

### DISCUSSION THREAD

**Discussion by ga2k** [utelle/wxsqlite3#90] [Ideas]
Hey there, I was wondering if you have any plans to make this library available via vcpkg?

Although it is simple thing to clone the git and rebuild it, having it as an auto-dependency using vcpkg would be great. In an app I am currently coding, all libraries except this and mailio [https://github.com/karastojko/mailio](url) are available there, and that makes moving between machines a breeze.

Thanks for your consideration.

**Comment by utelle** [comment]
Well, you mentioned this idea already quite some time ago in another issue https://github.com/utelle/wxsqlite3/issues/72#issuecomment-675187462. Be assured that I haven't forgotten it. However, I was distracted from this _vcpkg_ topic for various reasons.

I intend to add _vcpkg_ support in the future, but I don't have a date yet, when this will happen.
